add new column with pand_ids per VBO_ID

PeeWeeOSM opened the pull request:
add new column with pand_ids per VBO_ID (gerelateerde_panden_dit_VBO) to final table

PeeWeeOSM commented:
See issue #325

justb4 reviewed (commented):
Thanks for this @PeeWeeOSM ! This is a long-lasting and hard issue to solve. Some suggestions:

* the new column is of type string with comma separated PND id's. Would using a string array as in original VBO table not be a better choice?
* there are some other scripts dependent on the adres-plus table, like `adres-plus2csv.sql` and to generate the other adres-tables, like `adres-full`. Should these also be adapted? (as `adres-tabel-full.sql` is in fact obsolete).

But ok, the current implementation does not affect existing applications. Maybe it is good to analyse what the main uses are of more PNDs to a VBO? "Doorgebroken muren"? This also affects total square meters, building years, status PND etc. 
A hard problem, at least when when exporting CSV. 

We could also say: "This is best effort". We at least provide the BAG id's. Advanced apps can consequently use the whole BAG model (tabels, views).

PeeWeeOSM commented:
@justb4 

- I am not an array-guy and see no advantage in using that.
- I am not familiar with "adres-tabel-full.sql" so no idea who would want this extra column added. I have added this column to adres-plus2csv.sql.  

I have now added a new column in adres_plus but I could have altered existing column "pand_id" with this new value. A new column does not break anything but it all depends on use cases and I do not know what these all are. So I am fine with either solution.

I thought you were the one that wanted all thes pand_ids but it  could be that someone else asked you for this ;)

Regarding " total square meters, building years, status PND".  If there are request for this it could also be added but I have not seen any request for this. I personally do not really care.

justb4 reviewed (approved):
Ok, let's merge this PR and see later when new requirements come up.
